scripts/test-one.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+HOME=`dirname $(readlink -f "$0")`
+TEST_DIR=$1
+
+if [ -z "$TEST_DIR" ]; then
+    TEST_DIR="."
+elif [ ! -d "$TEST_DIR" ]; then
+    echo "${TEST_DIR} is not a directory. Defaulting to ."
+    TEST_DIR="."
+fi
+
+cd "$TEST_DIR"
+echo "==== Testing directory '$TEST_DIR'. ===="
+npm install
```

Comment:
Using yarn here would probably speed up the ci significantly since yarn is about 4 times faster then npm. And yarn deterministic in the way it installs dependencies which makes debugging much easier if we have to.

Reply:
I changed it but I never committed it. I think we talked about it literally yesterday. :)